apps/src/studio/levels.js

```diff
@@ -2381,6 +2359,9 @@ levels.js_hoc2015_event_random_items = {
       }
     }
   ],
+  "requiredForSuccess" : {
+    "setBotSpeed": true
```

Comment:
Ideally there will be a way to detect that this is run inside the right event handler, but I'm not yet sure how to do that.